static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -52,6 +53,14 @@ function FeedbackOnboardingSidebar(props: CommonSidebarProps) {
     allPlatforms: feedbackOnboardingPlatforms,
   });
 
+  useEffect(() => {
+    // this tracks clicks from any source: feedback index, issue details feedback tab, banner callout, etc
+    trackAnalytics('feedback.list-view-setup-sidebar', {
+      organization,
+      platform: currentProject?.platform ?? 'other',
```

Comment:
When could we end up with `null`? Since `other` is an actual platform (and unfortunately large % if it is Dart/Flutter now, how about we go specific and say:


```suggestion
      platform: currentProject?.platform ?? 'unknown',
```

Reply:
so the platform is from the project ID which is fetched from the API, which could always return `undefined` -- but i can update this result to say `unknown`!